modules/ROOT/pages/index.adoc

```diff
@@ -3,5 +3,67 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Anypoint Service Mesh Beta program was launched in October 2019. 
+:page-aliases: service-mesh-overview-and-landing-page.adoc
```

Comment:
If this was never in production, there is no need for a page-alias

Reply:
Ok! Thanks! Done!